src/Time/StepChoosers/StepChooser.hpp

```diff
@@ -110,7 +103,54 @@ class StepChooser : public PUP::able {
 
   WRAPPED_PUPable_abstract(StepChooser);  // NOLINT
 
-  using creatable_classes = Registration::registrants<StepChooserRegistrars>;
+  /// The `last_step_magnitude` parameter describes the slab size to be
+  /// adjusted; It may be infinite if the appropriate size cannot be determined.
+  ///
+  /// As opposed to the decision made by `desired_step` in the
+  /// StepChooser<StepChooserUse::LtsStep> specialization, the slab
```

Comment:
the class `StepChooser<StepChooserUse::LtsStep>` could be in backticks.

Reply:
It doesn't link correctly in backtics for some reason.  (It goes to the template documentation instead of the specialization.)